cider/datastore.py

```diff
@@ -341,12 +339,17 @@ def merge(self) -> None:
         # Make the smallest weight 1
         self.weights = self.merged['weight'] / self.merged['weight'].min()
 
-    def load_data(self, data_type_map: Mapping[DataType, Optional[Union[SparkDataFrame, PandasDataFrame]]]) -> None:
+    def load_data(
+        self, 
+        data_type_map: Mapping[DataType, Optional[Union[SparkDataFrame, PandasDataFrame]]],
+        all_required = True
```

Comment:
prefer type annotations even for params with default values for consistency with the rest of the function signature